gnmidiff/setrequest.go

```diff
@@ -212,26 +212,54 @@ func (intent *setRequestIntent) populateUpdate(pathToLeaf string, val interface{
 	return nil
 }
 
+// prefixStr returns the path version of a prefix path, handling corner cases.
+func prefixStr(prefix *gpb.Path) (string, error) {
+	if prefix == nil {
+		prefix = &gpb.Path{}
```

Comment:
`return "", nil` instead

Reply:
Actually this is intentional -- it's to make sure prefix is empty (but not nil) so that `ygot.PathToString` doesn't error out.

Reply:
Done.